### ISSUE TITLE
Define Quadrants

### ISSUE THREAD

**Issue by grahamar** [flowcommerce/tech-radar#1]
We should define what quadrants we want to have:

[ ] - Languages
[ ] - Frameworks
[ ] - Infrastructure
[ ] - Tools
[ ] - Data management
[ ] - Techniques

Non-Exhaustive Examples:
- https://opensource.zalando.com/tech-radar/
- https://www.thoughtworks.com/radar
- https://github.com/gilt/standards/tree/master/recommendations
- https://atlassian.github.io/techradar/?sheetId=1YXkrgV7Y6zShiPeyw4Y5_19QOfu5I6CyH5sGnbkEyiI
- https://xt-techradar.com/

**Comment by dasmb** [comment]
Prefer to combine `languages` with another quadrant (either tools or techniques)

**Comment by grahamar** [comment]
Yeah I've seen "Languages & Frameworks" combined a lot. Since they're generally tightly coupled
